### PR TITLE
Kuyawa universe

### PR DESCRIPTION
**Pull request by kuyawa**
Profile form

### CODE REVIEW

File: packages/blockchain-tools/src/contracts/solidity/erc6551/erc1155-abi.json
Comment (JulianKingman) [inline packages/blockchain-tools/src/contracts/solidity/erc6551/erc1155-abi.json:1]: Why is erc1155 in the erc6551 folder? Are the accounts attached to erc1155's? I thought they attach to 721's

Comment (JulianKingman) [reply packages/blockchain-tools/src/contracts/solidity/erc6551/erc1155-abi.json:1]: Also if these are just regular erc1155 definitions, could we put them in `solidity/erc1155`? Sorry if this is a dumb question.

File: packages/blockchain-tools/src/contracts/solidity/erc6551/erc721-abi.json
Comment (JulianKingman) [inline packages/blockchain-tools/src/contracts/solidity/erc6551/erc721-abi.json:1]: Same question as with erc1155

File: packages/blockchain-tools/src/interfaces/FreighterWallet.ts [hunk 1]
```diff
@@ -58,20 +57,26 @@ class FreighterWallet extends ChainBaseClass {
...
     try {
-      console.log("From", this.connectedWallet)
-      console.log("Paying", amount, "XLM to", address, "Memo", memo)
+      console.log("From", this.connectedWallet || '???')
+      console.log("Paying", amount, "XLM to", address, "Memo", memo || '[no]')
       const act = await this.horizon.loadAccount(this.connectedWallet)
+      //const fee = 5000
       const fee = await this.horizon.fetchBaseFee() // 100
-      const opr = StellarSDK.Operation.payment({
+      const xlm = this.fromBaseUnit(amount).toString()
```
Comment (JulianKingman) [inline packages/blockchain-tools/src/interfaces/FreighterWallet.ts:65]: We discussed the other day that amount should be the "large" unit and we should convert within the `sendPayment` function, did I understand right? In other words, FreighterWallet would be the one wallet that _wouldn't_ convert it.

File: packages/blockchain-tools/src/interfaces/Web3Server.ts [hunk 2]
```diff
@@ -40,12 +47,12 @@ export default class Web3Server extends ChainBaseClass {
     }
     const acct = this.web3.eth.accounts.privateKeyToAccount(walletSeed)
     const minter = acct.address
-    const instance = new this.web3.eth.Contract(Abi721, contractId)
+    const instance = new this.web3.eth.Contract(Abi721inc, contractId)
```
Comment (JulianKingman) [inline packages/blockchain-tools/src/interfaces/Web3Server.ts:50]: Do we still use the non-auto-incrementing 721? If not, we could get rid of it.

File: packages/types/src/Database.ts [hunk 3]
```diff
@@ -20,3 +20,29 @@ export const ReceiptStatus = {
...
+  organization: "organization",
+  initiative: "initiative",
+  event: "event",
+  impact: "impact",
+  story: "story",
+  user: "user",
+} as const
+
+// Used in contracts model
+export type ContractType = {
```
Comment (JulianKingman) [inline packages/types/src/Database.ts:36]: Love this 👍

File: turbo.json [hunk 4]
```diff
@@ -23,5 +23,8 @@
     "//#format-and-lint:fix": {
       "cache": false
     }
+  },
+  "remoteCache": {
```
Comment (JulianKingman) [inline turbo.json:27]: Is this important? The remote cache is a nice thing to have.

**Review by JulianKingman — COMMENTED** [review]
A few questions, mostly looks good 👍

File: packages/app-config/src/config/appConfigBase.ts [hunk 5]
```diff
@@ -30,10 +30,10 @@ const chains: AppConfig["chains"] = {}
 const auth: AppConfig["auth"] = []
 
 const chainDefaults: AppConfig["chainDefaults"] = {
-  network: "mainnet",
-  wallet: "xaman",
-  chain: "xrpl",
-  coin: "XRP",
+  network: "testnet",
```
[the remark this answers is not in the document]
Comment (kuyawa) [reply packages/app-config/src/config/appConfigBase.ts:33]: ok will revert it back to mainnet and inherit from there

File: packages/blockchain-tools/src/contracts/solidity/erc6551/erc1155-abi.json
Comment (kuyawa) [reply packages/blockchain-tools/src/contracts/solidity/erc6551/erc1155-abi.json:1]: I guess they got copied over from my test folder, I'll remove them

File: packages/blockchain-tools/src/interfaces/FreighterWallet.ts (hunk 1 above)
Comment (kuyawa) [reply packages/blockchain-tools/src/interfaces/FreighterWallet.ts:65]: Yes, we should convert in the sendPayment method, I don't know what that conversion does there, will remove it

File: packages/blockchain-tools/src/interfaces/Web3Server.ts (hunk 2 above)
Comment (kuyawa) [reply packages/blockchain-tools/src/interfaces/Web3Server.ts:50]: Yes, for TBAs we pass the entity ID as tokenid

**Review by JulianKingman — APPROVED** [review]

**Pull request merged**